TypeScript error: TS2309: An export assignment cannot be used in a module with other exported elements.

I try to import Penpal like this.
```
import * as Penpal from 'penpal';

The, I got error:
```
Error:(43, 3) TS2309: An export assignment cannot be used in a module with other exported elements.
```

In the following line of index.t.ds.
```
  export = Penpal;
```

When I import Penpal like this:
```
import Penpal from 'penpal';
```

I got error below:
```
Error:(3, 8) TS1192: Module ''penpal'' has no default export.
```

penpal 3.0.0 / typescript 2.6.2

Unfortunately not.

```
$ npm install -g typescript
$ mkdir test
$ cd test
$ npm init -y
$ git clone https://github.com/Aaronius/penpal
$ pushd penpal
$ npm install
$ npm run-script build
$ popd
$ cat > test.ts
import Penpal from './penpal';
console.log(Penpal);
$ tsc test.ts
penpal/lib/index.d.ts(33,21): error TS2693: 'Promise' only refers to a type, but is being used as a value here.
penpal/lib/index.d.ts(43,3): error TS2309: An export assignment cannot be used in a module with other exported elements.
test.ts(1,20): error TS2306: File '/private/tmp/test/penpal/lib/index.d.ts' is not a module.
```

It just works if I just remove index.d.ts ...

--- a/README.md
+++ b/README.md
@@ -20,6 +20,16 @@ And import Penpal into your code with something like:
 
 `import Penpal from 'penpal';`
 
+#### TypeScript
+
+Penpal provides a `types` file for usage with TypeScript; importing is simply the same as above:
+
+`import Penpal from 'penpal';`
+
+If importing with the error codes you will need to do something like:
+
+`import { default as Penpal, ERR_CONNECTION_DESTROYED } from 'penpal';`
+
 ### Using a CDN
 
 If you don't want to use npm to manage client packages, Penpal also provides a UMD distribution in a `dist` folder which is hosted on a CDN:

--- a/README.md
+++ b/README.md
@@ -22,13 +22,7 @@ And import Penpal into your code with something like:
 
 #### TypeScript
 
-Penpal provides a `types` file for usage with TypeScript; importing is simply the same as above:
-
-`import Penpal from 'penpal';`
-
-If importing with the error codes you will need to do something like:
-
-`import { default as Penpal, ERR_CONNECTION_DESTROYED } from 'penpal';`
+Penpal provides a `types` file for usage with TypeScript; importing is simply the same as above.
 
 ### Using a CDN
 

--- a/src/index.d.ts
+++ b/src/index.d.ts
@@ -13,6 +13,10 @@ declare namespace Penpal {
     [P in keyof T]: () => Promise<any>;
   };
 
+  type ERR_CONNECTION_DESTROYED = 'ConnectionDestroyed';
+  type ERR_CONNECTION_TIMEOUT = 'ConnectionTimeout';
+  type ERR_NOT_IN_IFRAME = 'NotInIframe';
+
   interface IConnectionOptions {
     methods?: ConnectionMethods<{}>;
     timeout?: number;
@@ -32,16 +36,16 @@ declare namespace Penpal {
     connectToParent(options?: IParentConnectionOptions): IConnectionObject;
     Promise: typeof Promise;
     debug: boolean;
-    ERR_CONNECTION_DESTROYED: string;
-    ERR_CONNECTION_TIMEOUT: string;
-    ERR_NOT_IN_IFRAME: string;
+    ERR_CONNECTION_DESTROYED: ERR_CONNECTION_DESTROYED;
+    ERR_CONNECTION_TIMEOUT: ERR_CONNECTION_TIMEOUT;
+    ERR_NOT_IN_IFRAME: ERR_NOT_IN_IFRAME;
   }
 }
 
 declare module 'penpal' {
   const Penpal: Penpal.PenpalStatic;
-  export = Penpal;
-  export var ERR_CONNECTION_DESTROYED: string;
-  export var ERR_CONNECTION_Timeout: string;
-  export var ERR_CONNECTION_DESTROYED: string;
+  export default Penpal;
+  export const ERR_CONNECTION_DESTROYED: Penpal.ERR_CONNECTION_DESTROYED;
+  export const ERR_CONNECTION_TIMEOUT: Penpal.ERR_CONNECTION_TIMEOUT;
+  export const ERR_NOT_IN_IFRAME: Penpal.ERR_NOT_IN_IFRAME;
 }

